spec/sec-properties-of-the-regexp-prototype-object-patch.html

```diff
@@ -161,4 +163,19 @@ <h1><ins>MakeIndicesArray ( _S_ , _indices_, _groupNames_ )</ins></h1>
       </emu-alg>
     </emu-clause>
   </emu-clause>
+
+  <emu-clause id="sec-get-regexp.prototype.indices">
+    <h1><ins>get RegExp.prototype.indices</ins></h1>
```

Comment:
This seems like an odd name - the other names describe the regular expression or how it matches, like “sticky”, “ignoreCase”, “multiline” etc - “indices” isn’t a verb or an adjective. “hasIndices” or something might make more sense?